docs/website/docs/plus/production/observability.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Observability
+description: Observability tooling
+keywords: [observability, monitoring, alerting]
+---
+
+import Link from '../../_plus_admonition.md';
+
+<Link/>
+
+# Observability
```

Comment:
You could add a screenshot of the platform here, this would make this empty page look better.

Reply:
we decided we don't want to market all at once